src/scene/scene.cpp

```diff
@@ -453,41 +508,35 @@ scripting::LuaLibrary Scene::luaLibrary() {
                 "setPropertyValue",
                 &luascriptfunctions::property_setValue,
                 {},
-                "string, *",
+                "name, value [, duration, optimization]",
```

Comment:
"optimization" could be renamed to something that indicates that it has to do with regex

Reply:
Would be cool to allow for different easing options: Linear, EaseIn, EaseOut, EaseInOut.